CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and **Merged pull requests**. Critical items to know are:
 Referenced versions in headers are tagged on Github, in parentheses are for pypi.
 
 ## [vxx](https://github.com/pydicom/deid/tree/master) (master)
- - changing iteration technique through fields to properly add nested uids [#153](https://github.com/pydicom/deid/issues/153) (0.2.22)
+ - changing iteration technique through fields to properly add nested uids [#153](https://github.com/pydicom/deid/issues/153), change to return results from detect when recipe does not contain filters [#155](https://github.com/pydicom/deid/issues/155) (0.2.22)
```

Comment:
```suggestion
 - changing iteration technique through fields to properly add nested uids [#153](https://github.com/pydicom/deid/issues/153) (0.2.22) 
 - change to return results from detect when recipe does not contain filters [#155](https://github.com/pydicom/deid/issues/155)
```
These can generally be still on two lines, just with the version tag at the top of the list.